README.md

```diff
@@ -115,9 +115,26 @@ This section explains how to use Backstage:
 
 Before to install and use our Backstage application, it is needed to perform some steps such as:
 - Create an OpenShift project 
-- Provide your registry credentials (quay.io, docker, etc) as a `config.json` file
+- Provide your registry credentials (quay.io, docker, etc) as `config.json` file
+- Create a ServiceAccount for backstage (needed to get the token)
+- Create a ClusterRoleBinding for the SA to allow backstage to access the Kube API resources
+- Create the Fedora Podman VM using your public key to ssh
 
-The commands described hereafter will help you to set up what it is needed:
+The commands hereafter will guide you to set up what it is needed
+
+**Important**: Alternatively, you can also use our bash script: [provision-namespace.sh](scripts%2Fprovision-namespace.sh)
+```bash
+./scripts/provision-namespace.sh -h
+Processing -h
+Usage: ./scripts/provision-namespace.sh [options]
+Options:
+  -n, --namespace     <namespace>                        The namespace on the QShift cluster (mandatory)
+  -q, --quay          <quay_username:quay_password>      The Quay username and password (mandatory)
+  -o, --quay-org      <quay_username:quay_password>      The Quay organization hosting the images (mandatory)
+  -d, --docker        <docker_username:docker_password>  The docker username and password (mandatory)
+  -k, --key-path      <public_key_path>                  The path of your public to ssh to the VM (mandatory)
+  -r, --dry-run                                          Run the kubectl command with dry-run=client
+```
 
```

Comment:
```suggestion
For example:

./scripts/provision-namespace.sh -n mynamespace -q "myquayuser:myquaypwd" -o "myquayorg:myquaypwd" -d "mydockeruser:mydockerpwd" -k $HOME/.ssh/id_rsa.pub
```

Reply:
There is mistake with your example. That should be 
```
./scripts/provision-namespace.sh -n my-namespace -q "my-quay-user:my-quay-pwd" -o "my-quay-org" -d "my-docker-user:my-docke-rpwd" -k $HOME/.ssh/id_rsa.pub
```

Reply:
ok, I have replaced my suggestion, I think you can commit it.

Reply:
Still a typo mistake "my-docke-rpwd" -> my-docker-pwd"

Reply:
done

Reply:
Your commit suggestion is still 
```
./scripts/provision-namespace.sh -n my-namespace -q "my-quay-user:my-quay-pwd" -o "my-quay-org" -d "my-docker-user:my-docke-rpwd" -k $HOME/.ssh/id_rsa.pub
```
and not 
```
./scripts/provision-namespace.sh -n my-namespace -q "my-quay-user:my-quay-pwd" -o "my-quay-org" -d "my-docker-user:my-docker-pwd" -k $HOME/.ssh/id_rsa.pub
```

Reply:
I think it's ok, if not you can apply the correct one from your side.